sysusers.d/systemd-network.conf.in

```diff
@@ -0,0 +1,10 @@
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+{% if ENABLE_NETWORKD %}
```

Comment:
Also here.

Reply:
Right, there doesn't need to be a .in pre-processing anymore.

Reply:
argh, for networkd, you still need .in pre-processing for the UID, but no condition, as @yuwata said.

So disregard what I said. I was looking at systemd-oom.conf.in when commenting.

Reply:
> argh, for networkd, you still need .in pre-processing for the UID, but no condition, as @yuwata said.

So the `{% if ENABLE_NETWORKD %}` around is needed for the inner `{{SYSTEMD_NETWORK_UID}}` to work?

Reply:
No.